src/test/esm-loader.spec.ts

```diff
@@ -358,6 +359,88 @@ test.suite('esm', (test) => {
         });
       }
 
+      test.suite('esm child process working directory', (test) => {
+        test('should have the correct working directory in the user entry-point', async () => {
+          const { err, stdout, stderr } = await exec(
+            `${BIN_PATH} --esm --cwd ./working-dir/esm/ index.ts`
+          );
+
+          expect(err).toBe(null);
+          expect(stdout.trim()).toBe('Passing');
+          expect(stderr).toBe('');
+        });
+
+        test.suite(
+          'with NodeNext TypeScript resolution and `.mts` extension',
```

Comment:
Is this `.mts` variant still required?  I see that the previous test case used regexp matching to avoid `import.meta.url`.  But if we can avoid that issue via `transpileOnly`, then I think this test case is identical to the previous one?

Reply:
Likely yes. Although these tests are still a little different:

* One loads the worker through an absolute path
* One loads the worker relatively